### ISSUE TITLE
How can I pretrain the model to get a good intial point for softmax loss?

### ISSUE THREAD

**Issue by PKUWison** [ShuangLI59/person_search#33]
Hello Shuang，
In your work, you have compared the OIM loss with the softmax loss and softmax + pretrain. Obviously, your work is perfect. However, how do you pretrain the model, if you just use the softmax loss? Actually I have read your paper in ArXiv2016, which introduced the process of pretrain. So I try to pretrain the model in CVPR2017 with the help of the work in ArXiv2016. I can't obtain the result in paper which the test mAP should be about 60%. I just obtain 36% test mAP. I really don't konw the reason. Can you tell me the difierences of the pretrain model and the train model. And how should I solve the model with only softmax+pretrain? Thanks you so much! Have a nice day!

**Comment by Cysu** [comment]
We just use an ImageNet-pretrained ResNet-50 and replace its last FC layer by a new one with 5533 units (including background class). You may refer to our [v1](https://github.com/ShuangLI59/person_search/tree/v1) branch as an example, although it uses VGG16 rather than ResNet-50.

Note that the LMDB for pretraining contains half of the data as randomly sampled background regions. And the learning rate and the softmax weights should be carefully tuned.

You may also email me your pretraining log, so that I could better understand your settings.

**Comment by PKUWison** [comment]
Thank a lot for your prompt reply! I will email you my pretraining log at once！
